files/usegalaxy_main_tool_list.yaml

```diff
@@ -1696,7 +1696,27 @@ tools:
   tool_panel_section_label: 'Text Manipulation'
   tool_shed_url: https://toolshed.g2.bx.psu.edu/
   install_resolver_dependencies: False
-
+- name: tsne
+  owner: moheydarian
+  revisions:
+  - c99a60c49682
+  tool_panel_section_label: 'Graph/Display Data'
+  tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
```

Comment:
Yes, I've got a number of tools that are works in progress and am keeping them in TTS for now.